properties.yaml

```diff
@@ -167,6 +167,15 @@ DevToolsTransparencyEnabled:
     Some platforms might not be able to render transparency correctly (e.g. some linux environments).
     This property will allow such platforms to disable/enable transparency
 
+DevToolsAlwaysOnTop:
+  key: compose.reload.devToolsAlwaysOnTop
+  type: boolean
+  default: "true"
```

Comment:
It should default to false in my opinion. It's confusing if the app window is behind another window, but you can see the tool window next to "nothing" regardless.